.travis.yml

```diff
@@ -1,17 +1,12 @@
 language: csharp
 sudo: false
-mono:
-  - 4.6.2 # in most common distros
-  - latest
-os:
-  - linux
-  - osx
 matrix:
-  exclude:
-    - os: osx
+  include:
+    - os: linux
+      mono: 4.6.2
+    - os: linux
       mono: latest
-  allow_failures:
-    - mono: latest
-  fast_finish: true
+    - os: osx
+      mono: 4.6.2
```

Comment:
I have no OSX access either. 😞 I had a Google though - and `brew install mono` currently gets 5.0.1.1 - it looks like it's kept fairly up to date. While we're here - can I suggest just test on OSX latest, rathere than just 4.6.2?